apps/docs/self-hosting/configuration-options.mdx

```diff
@@ -68,3 +68,56 @@ These variables need to be configured to let Rallly send out transactional email
 <ParamField path="SMTP_TLS_ENABLED" default={"false"}>
   Enable TLS for your SMTP connection
 </ParamField>
+
+### Openid Connect Configuration
+
+These variables needs to be configured to enable oauth 2.0 or openid connect (oidc) log in.
+Rallly use [NextAuth.js](https://next-auth.js.org/) for authentication and most options are derived from the framework.
```

Comment:
Next-auth is an implementation detail that does not need to be exposed to self-hosters. Can you remove these lines? These docs should be self-sufficient or at most point to docs that are referring to open standards like oauth 2.0 or openid.